.gitignore

```diff
@@ -2,6 +2,8 @@ node_modules
 /out
 src/raw/bower_components
 tmp
+deploy-key
```

Comment:
Well, `deploy-key` and `deploy-key.pub` should never be commited to repo, so this is to make sure that someone doesn't commit them by accident.